azure/packages/test/end-to-end-tests/src/test/containerCreate.spec.ts

```diff
@@ -126,7 +126,7 @@ describe("Container create scenarios", () => {
 	 *
 	 * Expected behavior: an error should be thrown when trying to get a non-existent container.
 	 */
-	it("cannot load improperly created container (cannot load a non-existent container)", async () => {
+	it.skip("cannot load improperly created container (cannot load a non-existent container)", async () => {
```

Comment:
```suggestion
	it.skip("cannot load improperly created container (cannot load a non-existent container)", async () => {
```
can we add some `TODO` note above this skipping tests